modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -187,27 +186,31 @@ def mask(self, row_labels, col_labels):
         PandasDataframePartition
             New `PandasDataframePartition` object.
         """
-
-        def is_full_axis_mask(index, axis_length):
-            """Check whether `index` mask grabs `axis_length` amount of elements."""
-            if isinstance(index, slice):
-                return index == slice(None) or (
-                    isinstance(axis_length, int)
-                    and compute_sliced_len(index, axis_length) == axis_length
-                )
-            return (
-                hasattr(index, "__len__")
-                and isinstance(axis_length, int)
-                and len(index) == axis_length
-            )
+        # FIXME: for a `take` won't is_full_axis_mask be wrong if row_labels
+        #  is non-unique or not monotonic increasing.
+
+        # def is_full_axis_mask(index, axis_length):
```

Comment:
It's not very clear to me why it's commented out. Can you add details or rephrase?